src/components/ComponentSamples/ComponentSamplesPage.tsx

```diff
@@ -103,14 +110,55 @@ export const ComponentSamplesPage = () => {
       footer={footer}
     >
       <PageSection padding={{ default: 'noPadding' }} isFilled>
-        <StatusBox
-          skeleton={skeletonCatalog}
-          data={items}
-          loaded={loaded}
-          loadError={loadError}
-          label="Catalog items"
-        >
-          <CatalogView items={items} renderTile={renderTile} hideSidebar={true} />
+        <StatusBox data={items} loaded={loaded} loadError={loadError} label="Catalog items">
+          <Toolbar>
```

Comment:
For consistent spacing around the toolbar.

```suggestion
          <Toolbar usePageInsets>
```